01.fizzbuzz/fizzbuzz.rb

```diff
@@ -0,0 +1,10 @@
+(1..20).each do |i|
+  if i % 15 == 0
+    puts "FizzBuzz"
+  elsif i % 3 == 0
+    puts "Fizz"
+  elsif i % 5 == 0
+    puts "Buzz"
+  else puts i
```

Comment:
```suggestion
  else
    puts i
```

誤解の原因になるので改行した方がいいですね

参考: 似たような話を昔Qiitaに書いたことがあります
https://qiita.com/jnchito/items/ef05b88657019f434df9